src/utils/HistoryCaseRevision.functions.js

```diff
@@ -1,12 +1,33 @@
 const functions = {
   markComplete: (id, prevState) => {
-    if (!prevState.markSchemeElements[id])
+    console.log("this is id", id);
```

Comment:
Do these console logs still need to be here?

Reply:
done